charts/shoot-core/components/charts/nodelocal-dns/templates/configmap.yaml

```diff
@@ -2,10 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "node-local-dns.config.name" . }}
+  name: node-local-dns-configDataHash
```

Comment:
What's this/why did you add this suffix? I would suggest to just drop the entire file since the name of the `ConfigMap` is unique anyways, i.e. it will be garbage collected (the same was done in #5056).